PhysicsTools/PatAlgos/python/producersLayer1/electronProducer_cfi.py

```diff
@@ -63,16 +63,8 @@
     userIsolation = cms.PSet(),
 
     # electron ID
-    addElectronID = cms.bool(True),
-    electronIDSources = cms.PSet(
-        # configure many IDs as InputTag <someName> = <someTag> you
-        # can comment out those you don't want to save some disk space
-        eidRobustLoose      = cms.InputTag("eidRobustLoose"),
-        eidRobustTight      = cms.InputTag("eidRobustTight"),
-        eidLoose            = cms.InputTag("eidLoose"),
-        eidTight            = cms.InputTag("eidTight"),
-        eidRobustHighEnergy = cms.InputTag("eidRobustHighEnergy"),
-    ),
+    addElectronID = cms.bool(False),
+    electronIDSources = cms.PSet(),
```

Comment:
@arizzi @gpetruc 
is somebody still using the default PAT?

Instead of moving more common stuff into PAT individual configuration files, the "gravity" pulls everything into miniAOD_tools.py